Makefile

```diff
@@ -49,14 +49,18 @@ RES_FILES = $(shell find $(RES_DIR) -type f)
 # Debug signing
 
 DEBUG_KEYSTORE = _build/debug.keystore
+DEBUG_KEYSTORE_ASC = _build/debug.keystore.asc
 DEBUG_PASSWD = debug0
 
+$(DEBUG_KEYSTORE_ASC):
+	gpg -c --armor --output "$@" --pinentry-mode loopback --passphrase debug0 --yes $(DEBUG_KEYSTORE)
```

Comment:
This rule isn't doing something more than just encode the keystore in a different format. I think this shouldn't be in the Makefile, the best place would be in documentation explaining how the github secret is used (perhaps in Contributing.md)

Reply:
I thought that was very convenient ( a good thing ), can we at least have that command inside makefile, but instead on a separate phony rule, so it is not a requirement for the debug.apk ?

If you really say that the way to go is to remove that, and put that on documentation, so the dev working on the app needs to type the whole command manually, that is okay too. I can remove that. But honestly I was really happy with the way it is right now, the most convenient.

Reply:
I've added documentation for the way it is implemeted currently:  3912d8b6d90ece5279e13b5a42c0389e19951241
https://github.com/sdrapha/Unexpected-Keyboard/blob/all-branches-merged-in-one/CONTRIBUTING.md#contributing=

If you decide you want the other way, that documentation also needs adjustments.

What do you think?

Reply:
Yes, that's what I meant, remove the rule from the Makefile and expect people to copy it from the documentation. I don't like that it's a requirements to the apk and that's it's present in the Makefile at all. A PHONY rule wouldn't be useful, it's not much more complicated to type the full command than to type the make command.

Reply:
ok, I'll remove from the makefile then, and adjust the documentation accordingly